docs/app/examples/additional-map-options.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+angular.module('esri-map-docs')
+    .controller('AdditionalMapOptionsCtrl', function($scope, esriLoader) {
+        $scope.map = {
+            center: {
+                lng: -96.53,
+                lat: 38.374
+            },
+            zoom: 13,
+            mapOptions: {
+                minZoom: 10,
+                maxZoom: 15,
+                resizeDelay: 500,
+                navigationMode: 'classic',
+                sliderOrientation: 'horizontal',
+                sliderPosition: 'top-right',
+                displayGraphicsOnPan: true,
+                fadeOnZoom: false,
+                logo: false,
+                wrapAround180: false
+            }
+        };
+
+        /*esriLoader.require(['esri/geometry/Extent', 'esri/SpatialReference'], function(Extent, SpatialReference) {
```

Comment:
@tomwayson thank you for reviewing what is in here so far! You're completely right about the commented out code showing up in the front-facing code snippet.  That's just sitting in there temporarily until I move it out to a new example/test page that loads in an extent and infoWindow from the mapOptions.